extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -55,16 +55,34 @@ amp-carousel[controls] .amp-carousel-button,
 
 .amp-carousel-button-prev {
   left: 16px;
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="#fff" viewBox="0 0 18 18"><path d="M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z"/></svg>');
-  background-size: 18px 18px;
 }
 
 .amp-carousel-button-next {
   right: 16px;
+}
+
+.i-amphtml-carousel-button-prev-legacy {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" fill="#fff" viewBox="0 0 18 18"><path d="M15 8.25H5.87l4.19-4.19L9 3 3 9l6 6 1.06-1.06-4.19-4.19H15v-1.5z"/></svg>');
```

Comment:
You could actually just leave these in the rule above, since the new versions would override them due to selector ordering (as they have the same specificity).

Reply:
Hmm, so do you plan on keeping an internal class after the migration is done or just for now? Either way is fine though.

Reply:
I intend on keeping the internal classes - makes it more easier to grep. Matches with other components too that way.